tests/robot-tests/tests/libs/utilities.py

```diff
@@ -223,13 +223,13 @@ def capture_large_screenshot():
 def user_checks_previous_table_tool_step_contains(step, key, value):
     try:
         sl.wait_until_page_contains_element(
-            f'xpath://*[@id="tableToolWizard-step-{step}"]//*[text()="Go to this step"]')
+            f'xpath://*[@id="tableToolWizard-step-{step}"]//*[text()="Go to this step"]', timeout=30)
```

Comment:
So this worked?

Would refactor this so that you're passing the timeout in as a default argument to this function.

Reply:
Not sure if it worked - I don't think the failure has occurred against my branch since I added it